src/pyOpenMS/addons/MatrixDouble.pyx

```diff
@@ -89,7 +56,7 @@ from Matrix cimport *
         cdef int j = 0
         for i in range(int(rows)):
             for j in range(int(cols)):
-                mat_.setValue(j,i,data[i][j])
+                mat_.setValue(i,j,data[i][j])
```

Comment:
@timosachsenberg in your example this was switched. Any reason?

Reply:
Hmm don’t recall. Maybe I mixed it up with how eigen indexes a matrix

Reply:
hmm I think I copied your example and its currently incorrect. That is why I wrote the test, according to the test it is correct now :-)